### PR TITLE
Pkg-lists

### PR DESCRIPTION
**Pull request by memsharded**
Docs for https://github.com/conan-io/conan/pull/13928

### CODE REVIEW

File: introduction.rst [hunk 1]
```diff
@@ -88,7 +88,7 @@ There are some things that are not included in this commitment:
 - Builtin default implementation of extension points as plugins or hooks can also change with every release. Users can provide their own ones for stability.
 - Output of packages templates with ``conan new`` can update at any time to use latest features.
 - The output streams stdout, stderr, i.e. the terminal output can change at any time. Do not parse the terminal output for automation.
-- Anything that is explicitily labeled as ``experimental`` or ``preview`` in the documentation, or in the Conan cli output.
+- Anything that is explicitily labeled as ``experimental`` or ``preview`` in the documentation, or in the Conan cli output.Read below section for a detailed definition of these labels.
```
Comment (RubenRBS) [inline introduction.rst:91]:
```suggestion
- Anything that is explicitly labeled as ``experimental`` or ``preview`` in the documentation, or in the Conan cli output. Read the section below for a detailed definition of these labels.
```

File: tutorial/other_features.rst [hunk 2]
```diff
@@ -9,3 +9,18 @@ python_requires
 It is possible to reuse code from other recipes using the :ref:`python_requires feature<reference_extensions_python_requires>`.
 
 If you maintain many recipes for different packages that share some common logic and you don't want to repeat the code in every recipe, you can put that common code in a Conan ``conanfile.py``, upload it to your server, and have other recipe conanfiles do a ``python_requires = "mypythoncode/version"`` to depend on it and reuse it.
+
+Packages lists
+--------------
+
+It is possible to manage a list of packages, recipes and binaries together with the "packages-list" feature. 
+Several commands like ``upload``, ``download``, ``remove`` allow receiving a list of packages file as an input, and they can do that operation over that list.
```
Comment (RubenRBS) [inline tutorial/other_features.rst:17]:
```suggestion
Several commands like ``upload``, ``download``, and ``remove`` allow receiving a list of packages file as an input, and they can do their operations over that list.
```

**Review by RubenRBS — APPROVED** [review]
Nice, some grammar typos and a few wording suggestions, but otherwise reads great. Seeing it like this really shows how useful this feature can be for the right users :)

File: examples/commands/pkglists.rst [hunk 3]
```diff
@@ -0,0 +1,176 @@
+.. _examples_commands_pkglists:
+
+Using packages-lists
+====================
+
+.. include:: ../../common/experimental_warning.inc
+
+
+Packages lists is a powerful and convenient Conan feature that allows to automate and concatenate different Conan commands.
```
[the remark this answers is not in the document]
Comment (franramirez688) [reply examples/commands/pkglists.rst:9]: Then, I think it'd be better to put it as:
```suggestion
``Packages lists`` is a powerful and convenient Conan feature that allows us to automate and concatenate different Conan commands.
```
Or:

```suggestion
``PackagesLists`` is a powerful and convenient Conan feature that allows us to automate and concatenate different Conan commands.
```

File: examples/commands/pkglists.rst [hunk 4]
```diff
@@ -0,0 +1,176 @@
+.. _examples_commands_pkglists:
+
+Using packages-lists
```
Comment (franramirez688) [inline examples/commands/pkglists.rst:3]: Use the same name for the title and the rest of the mentions.

**Review by czoido — APPROVED** [review]

**Comment by memsharded** [comment]
Good, I will do those changes later, when I add the ``remove --list`` docs.

**Pull request merged**